spec/requests/api/moves_controller_filter_spec.rb

```diff
@@ -13,16 +13,12 @@
   let(:expected_moves) { nil }
   let(:unexpected_moves) { nil }
 
-  shared_examples 'it returns the correct moves' do
+  shared_examples 'an api that filters moves correctly' do
     let(:size) { response_json['data'].size }
     let(:ids) { response_json['data'].map { |move| move['id'] } }
 
-    it 'contains all the expected moves' do
+    it 'contains the correct moves' do
       expect(ids).to match_array(expected_moves.pluck(:id))
-    end
-
-    it 'does not contain any unexpected moves' do
```

Comment:
I've combined these two tests into one to half the amount of request/response cycles we do